gitlab-logger.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+  name: gitlab-logger
+  version: 3.0.0
+  epoch: 0
+  description: GitLab Logger provides a means of wrapping non-structured log files within structure JSON.
+  copyright:
+    - license: MIT
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - go
```

Comment:
(not blocking) Both of these should come in from depending on `go/build`, so you could just remove this.